TreeList: Expand expandedRowKeys Info (#8309)
DevExpress/devextreme-documentation: concepts/05 UI Components/TreeList/60 Expand and Collapse Rows/Expand and Collapse Rows.md

> TreeList allows you to configure expanded rows on initialization, as well as expand/collapse rows programmatically.

Reviewer: ```suggestion
TreeList allows you to configure expanded rows on initialization, and expand/collapse rows programmatically.
```
